README.rst

```diff
@@ -33,6 +33,16 @@ Our current main objective is the development of a new 3d octree game engine.
 
 Inexor is licensed under the `MIT license <https://github.com/inexorgame/vulkan-renderer/blob/master/LICENSE>`__.
 
+
+Sponsors
+########
+
+Special thanks to `JetBrains <https://www.jetbrains.com/?from=github.com/inexorgame>`__ for providing us with CLion licenses!
+
+
+.. image:: documentation/source/images/icon_CLion.svg
```

Comment:
Add a size limit otherwise we get a fullscreen logo.

Reply:
Done.